airlock/views/workspace.py

```diff
@@ -245,6 +250,45 @@ def multiselect_add_files(request, multiform, workspace):
     )
 
 
+def multiselect_update_files(request, multiform, workspace):
+    files_to_add = []
+    files_ignored = {}
+
+    # validate which files can be added
+    for f in multiform.cleaned_data["selected"]:
+        workspace.abspath(f)  # validate path
+
+        if policies.can_update_file_on_request(workspace, UrlPath(f)):
+            files_to_add.append(f)
+        else:
+            files_ignored[f] = "cannot add using the update dialogue"
```

Comment:
Should this check `can_add_file_to_request`?  If the file is already released, it won't be updatable, and this message implies it can be added, just not with this dialogue.

Reply:
I've done what I suggested above